djangocms_blog/admin.py

```diff
@@ -30,6 +31,59 @@ class EnhancedModelAdminMixin(object):
         pass
 
 
+# Bulk actions for post admin
+def make_published(modeladmin, request, queryset):
+    """ Bulk action to mark selected posts as published. If
+        the date_published field is empty the current time is
+        saved as date_published.
+    """
+    for post in queryset:
+        if not post.publish:
+            if not post.date_published:
+                post.date_published = timezone.now()
+            post.publish = True
+            post.save()
+
+
+def make_unpublished(modeladmin, request, queryset):
+    """ Bulk action to mark selected posts as UNpublished.
+    """
+    queryset.update(publish=False)
```

Comment:
I'd prefer emitting a message to confirm the action
Ditto for the actions below